src/Templates/Microsoft.NetCore.FSharp.ProjectTemplates.Web/source.extension.vsixmanifest

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.NetCore.FSharp.ProjectTemplates.Web" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>F# .NETCore Project Templates Test</DisplayName>
+    <Description>F# project templates for ASP.NETCore.</Description>
+  </Metadata>
+  <Installation SystemComponent="true" Experimental="true">
+    <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
```

Comment:
Are the express skus still a thing?  I'm pretty sure we ripped out our support for them.

Reply:
The express SKUs still ship.  I do not know if F# is installed in any of them.